### DISCUSSION TITLE
Configured to make some blocks pass through chests

### DISCUSSION THREAD

**Discussion by ApliNi** [PurpurMC/Purpur#980] [__Dismissed__] [answered]
like this: https://www.spigotmc.org/resources/containerpassthrough.81890/
The author of this plugin has not updated the plugin to version 1.18 nor has a config file, it doesn't work with GLOW_ITEM_FRAME for 1.18, can you guys get it done?

*Use translation software, please forgive me if there are any mistakes*

**Comment by SageSphinx63920** [comment]
You may want to use an other plugin or [this](https://modrinth.com/mod/clickthrough) fabric mod. I dont think this is purpurs job

**Reply by ApliNi** [reply]
Thanks, I am looking for other plugins

**Comment by encode42** [comment, marked as answer]
As said previously, this is better done with a plugin or client-side mod. I've been using the linked mod for over a year with no issue.

**Reply by ApliNi** [reply]
Should I turn it off? I don't know what to do here

**Reply by encode42** [reply]
Turn what off?

If you're confused about what to do about this discussion post, it's been archived. Nothing else needs to be done.

**Reply by ApliNi** [reply]
Okay, thanks)
